src/pydvl/influence/frameworks/torch_differentiable.py

```diff
@@ -67,7 +72,8 @@ def solve_linear(
     matrix = model.hessian(
         all_x, all_y, progress=progress
     ) + hessian_perturbation * identity_tensor(model.num_params, device=model.device)
-    return torch.linalg.solve(matrix, b.T).T
+    info = {"hessian": matrix}
```

Comment:
Not sure here but are we interested in the original Hessian or the regularized one? In Fragile paper they computed eigenvalues on the original one.

Reply:
should be done